facade/nodeFacade_test.go

```diff
@@ -21,15 +21,38 @@ import (
 	"github.com/ElrondNetwork/elrond-go/process"
 	vmcommon "github.com/ElrondNetwork/elrond-vm-common"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
 //TODO increase code coverage
 
+type interf interface {
```

Comment:
delete this :) no need for that and for the next 20 lines

Reply:
pushed by mistake. removed